test/acceptance/member_test.exs

```diff
@@ -6,9 +6,9 @@ defmodule Community.Acceptance.MemberTest do
     navigate_to "/"
     click_on "MEMBER DIRECTORY"
     click_on "Add yourself to the directory"
-    fill_in "member_name", "Scott Summers"
-    fill_in "member_email", "cyclops@example.com"
-    fill_in "member_company_name", "X-men"
+    fill_in "member", "name", "Scott Summers"
+    fill_in "member", "email", "cyclops@example.com"
+    fill_in "member", "company_name", "X-men"
     click_role "member-save"
     assert String.contains?(visible_page_text, "Thank you for signing up!")
```

Comment:
I know this isn't part of the PR, but maybe this is a good opportunity to switch this to `=~`?